src/data/items.h

```diff
@@ -1283,7 +1283,13 @@ const struct Item gItems[] =
         .itemId = ITEM_ESCAPE_ROPE,
         .price = 550,
```

Comment:
If it is a key item, `price` needs to be 0. From `item_menu.c`: 

```c
void Task_ItemContext_Sell(u8 taskId)
{
    s16* data = gTasks[taskId].data;

    if (ItemId_GetPrice(gSpecialVar_ItemId) == 0)
    {
        CopyItemName(gSpecialVar_ItemId, gStringVar2);
        StringExpandPlaceholders(gStringVar4, gText_CantBuyKeyItem);
        DisplayItemMessage(taskId, 1, gStringVar4, BagMenu_InitListsMenu);
    }
```

No check is made for `importance` or `pocket`, meaning that you could sell this if it had a non zero price, even if it was a key item.